client/my-sites/stats/activity-log-item/activity-actor.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+
+export default class ActivityActor extends PureComponent {
+	static propTypes = {
+		actor: PropTypes.shape( {
```

Comment:
Omitted actor is now gracefully handled and no default is supplied. No `isRequired` makes sense to me here.